### DISCUSSION TITLE
Not working: Extract SQL from pasted source code

### DISCUSSION THREAD

**Discussion by frastr** [dbeaver/dbeaver#19730] [Q&A] [answered]
### Description

When i paste source code into SQL Editor or terminal, source code are not automatic transform into SQL.
In version 23.0.1 this worked. In version 23.0.2 not more.

Sample:

~~~python
        "select *",
        "    from table",
        "    order by 1, 2, 3",
~~~

~~~sql
select *
    from table
    order by 1, 2, 3
~~~

### DBeaver Version

Community Edition 23.0.2

### Operating System

Linux, Windows 10

### Steps to reproduce

Paste source code into SQL editor or terminal:

~~~python
        "select *",
        "    from table",
        "    order by 1, 2, 3",
~~~

**Comment by LonwoLonwo** [comment, marked as answer]
Hello @frastr 

You need to enable this setting back:

![2023-04-17 18_20_32-Preferences](https://user-images.githubusercontent.com/45152336/232533584-3b1d2c3d-50b0-46a6-8af7-6df1b106321f.png)

We disabled it by default since the last release.

**Comment by frastr** [comment]
Many thanks, it's work.
